jina/docker/helper.py

```diff
@@ -45,6 +45,10 @@ def credentials_file():
     return Path.home().joinpath('.jina').joinpath('access.yml')
 
 
+def docker_credentials_file():
+    Path.home().joinpath('.jina').mkdir(parents=True, exist_ok=True)
+    return Path.home().joinpath('.jina').joinpath('docker_access.yml')
```

Comment:
Why do we need to store the docker credentials locally? Who stores it?

Reply:
yes, they are not needed